packages/core/src/dialects/abstract/index.ts

```diff
@@ -458,7 +458,7 @@ export abstract class AbstractDialect {
       ifExists: false,
     },
     delete: {
-      modelWithLimit: false,
```

Comment:
Correct me if I'm wrong @ephys but I believe that we want the options to default to false, so we can limit the amount of negations in the code

Reply:
Only when it comes to parameters, i.e. `delete()` and `delete({ force: false })` should be equivalent

In the case of `supports`, it doesn't really apply